README.md

```diff
@@ -45,6 +45,29 @@ from undebt.examples.class_inherit_object import grammar, replace
 ```
 **(3)** Run `undebt` with `my_class_inherit_object.py` as the pattern
 ```bash
-undebt --pattern ./my_class_inherit_object.py --input <directory or file to undebt>
+undebt --pattern ./my_class_inherit_object.py <directory or file to undebt> ...
 ```
 If you are undebting a directory, you will also need to pass `-e py` to tell `undebt` to search for Python files in that directory. _For a complete command line example and the full command line help, see the [command line documentation](http://undebt.readthedocs.io/en/latest/cli.html)._
+
+## Tips
+
+Most of these will make use of
+[`xargs`](http://man7.org/linux/man-pages/man1/xargs.1.html).
+
+### Using with `grep`/`git grep` to find files
+
+```bash
+grep -l <search> | xargs undebt -p path/to/pattern.py
```

Comment:
probably add an example which uses find:

```
find -name '*.js' | xargs grep -l ... | xargs undebt ...
```
(or a more robust solution with either `-print0` + `-0` or the ugly `-exec` syntax)